src/ocm/components/clusterConfiguration/networkConfiguration/AvailableSubnetsControl.tsx

```diff
@@ -92,37 +93,45 @@ export const AvailableSubnetsControl = ({
     },
     [],
   );
+  const itemsIpv4Subnets = buildOptions(IPv4Subnets);
   return (
     <FormGroup
       label="Machine network"
       labelInfo={isDualStack && 'Primary'}
       fieldId="machine-networks"
-      isRequired
+      isRequired={isRequired}
     >
       <FieldArray name="machineNetworks">
         {() => (
           <Stack>
             {isDualStack ? (
               values.machineNetworks?.map((_machineNetwork, index) => {
                 const machineSubnets = index === 1 ? IPv6Subnets : IPv4Subnets;
+                const itemsSubnets = buildOptions(machineSubnets);
                 return (
                   <StackItem key={index}>
-                    <OcmSelectField
+                    <SubnetsDropdown
                       name={`machineNetworks.${index}.cidr`}
-                      options={buildOptions(machineSubnets)}
-                      isRequired={isRequired}
+                      items={itemsSubnets}
                       isDisabled={isDisabled}
+                      defaultValue={
+                        itemsSubnets.length > 1 ? itemsSubnets[1].label : itemsSubnets[0].label
+                      }
```

Comment:
This seems reduntant, isn't `useAutoSelectSingleAvailableSubnet` on line 84 doing the same thing?